crates/services/sync/src/import.rs

```diff
@@ -232,67 +227,56 @@ where
         .in_current_span()
         .await
     }
+}
 
-    async fn get_block_for_header(
```

Comment:
This function wasn't removed - it's just moved below

Reply:
maybe call this function get_sealed_blocks, since it returns a both the headers and all associated data?

Reply:
Yep, that makes sense. Will change.